xbmc/guilib/GUISpinControl.cpp

```diff
@@ -502,7 +502,24 @@ void CGUISpinControl::Process(unsigned int currentTime, CDirtyRegionList &dirtyr
 
 void CGUISpinControl::Render()
 {
-  if ( HasFocus() )
+  if (m_label.GetLabelInfo().font)
+  {
+    const float space = 5;
```

Comment:
maybe avoid the magic number?

Reply:
what do? move it on an unnamed namespace?

Reply:
seem used also on Process